### PR TITLE
hints for INACCESSIBLE_BOOT_DEVICE

### PR DESCRIPTION
**Pull request by marxencc**
Hi there,

as discussed in https://github.com/xcp-ng/xcp/issues/181 i added the INACESSIBLE_BOOT_DEVICE hints
feel free to change to your taste.

### CODE REVIEW

File: docs/guests.md [hunk 1]
```diff
@@ -293,6 +293,19 @@ Help is welcome to help us reconcile both procedures into one.
 
 **Note**: Also have a look at our [Troubleshooting Guide - Windows PV-Tools](troubleshooting.html#windows-agent-pv-tools).
 
+### VMs with INACCESSIBLE_BOOT_DEVICE error
+
+You can try to manually inject the missing drivers in recovery mode.
+
+* Get the "Drivers" folder from the XCP Tools installation path (C:\PROGRAM FILES...) - from another VM or install the tools somewhere else to get it.
```
Comment (olivierlambert) [inline docs/guests.md:300]: Could it work directly by using the ISO available on download from https://github.com/xcp-ng/win-pv-drivers ?

Eg https://github.com/xcp-ng/win-pv-drivers/releases/download/v8.2.2.200-RC1/XCP-ng.Windows.PV.Drivers.8.2.2.200-rc1.iso

File: docs/guests.md [hunk 2]
```diff
@@ -293,6 +293,19 @@ Help is welcome to help us reconcile both procedures into one.
...
+* Create an ISO-Image containing the "Drivers" folder (see <http://imgburn.com>) and mount that ISO-Image to your VM
+* Boot to recovery mode and use the command line and the tool "dism" (see [Microsoft Docs](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/add-and-remove-drivers-to-an-offline-windows-image)) to inject the drivers (specifically the xenbus and xenvbd drivers) - watch out for the drive letter of the Windows installation and the CD-Drive ('D' and 'E' in the following example):
+
+```
+    dism /image:d:\ /add-driver /driver:e:\Drivers\xenbus\x64\xenbus.inf
+    dism /image:d:\ /add-driver /driver:e:\Drivers\xenvbd\x64\xenvbd.inf
+```
```
Comment (olivierlambert) [inline docs/guests.md:307]: No need to use tab or space inside the comment block, so just:

```
dism /image:d:\ /add-driver /driver:e:\Drivers\xenbus\x64\xenbus.inf
dism /image:d:\ /add-driver /driver:e:\Drivers\xenvbd\x64\xenvbd.inf
```

File: docs/guests.md (hunk 1 above)
Comment (marxencc) [reply docs/guests.md:300]: The ISOs contain the Installer for x86 and x64 but we need the driver files like xenvbd.inf and xenvbd.sys - so its more easy to just get them from the installer or another installation ... i had a quick look but it seems the driver files are not to be found precompiled in the win-xenvbd repo.

Comment (olivierlambert) [reply docs/guests.md:300]: Okay so let's keep it that way then :)

File: docs/guests.md (hunk 2 above)
Comment (marxencc) [reply docs/guests.md]: yep - removed them

**Comment by olivierlambert** [comment]
Thanks for your contribution :+1:

**Comment by groliveira18** [comment]
In my case that didn't work. I had to attach the disk in another vm and delete the `xen*` dlls from the folder `C:\Windows\System32\drivers` After that I attached the disk to the original vm and OS worked.

I believe this is the best and fastest solution.

**Comment by olivierlambert** [comment]
Pinging @stormi

**Comment by stormi** [comment]
@groliveira18 As I'm unable to judge which solution is the best, I suggest to open a new PR that adds yours in addition the one contributed by @marxencc.

By the way @marxencc any opinion about it?

**Comment by marxencc** [comment]
I havent done much of this stuff lately but i remember from the good old 
p2v days that one was either stuck with problem
A) there is a storage driver but it is a wrong one and causes a problem
or
B) there is no compatible storage driver.
Depending on what actually causes the issue i guess both methods could 
work.
I think older versions of windows only kept the currently active storage 
driver installed while newer versions keep the default drivers for 
fallback.
(Or newer versions try fallback and older ones do not. - it made p2v a 
lot easier back then..)

**Comment by groliveira18** [comment]
@stormi  Sorry for the misunderstanding. My English is not very good (I'm Brazilian), but I wanted to say that this is the best solution for me.

**Pull request merged**